### ISSUE TITLE
[BUG] - Import of zip fails due to missing grouped

### ISSUE THREAD

**Issue by zierbeek** [mealie-recipes/mealie#2548]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [X] This issue can be replicated on the demo site (https://demo.mealie.io/)

### What is the issue you are experiencing?

mealie fails to import my recipe, exported from the same version.
[pasta-all-amatriciana-4.zip](https://github.com/mealie-recipes/mealie/files/12601699/pasta-all-amatriciana-4.zip)

### Steps to Reproduce

1) Upload recipe
2) fail

### Please provide relevent logs

```
2023-09-11T20:45:35.961017409+02:00 INFO:     172.16.2.223:57654 - "GET /api/app/about HTTP/1.1" 200 OK
2023-09-11T20:45:35.975314388+02:00 INFO:     172.16.2.223:57678 - "GET /api/app/about HTTP/1.1" 200 OK
2023-09-11T20:45:35.975835668+02:00 INFO:     172.16.2.223:57668 - "GET /api/app/about HTTP/1.1" 200 OK
2023-09-11T20:45:43.414256295+02:00 INFO:     172.16.2.223:44464 - "POST /api/recipes/create-from-zip HTTP/1.1" 500 Internal Server Error
2023-09-11T20:45:43.414832345+02:00 ERROR:    Exception in ASGI application
2023-09-11T20:45:43.414860485+02:00 Traceback (most recent call last):
2023-09-11T20:45:43.414868833+02:00   File "/app/mealie/db/models/_model_utils/helpers.py", line 42, in safe_call
2023-09-11T20:45:43.414875613+02:00     return func(**get_valid_call(func, dict_args))
2023-09-11T20:45:43.414881986+02:00 TypeError: __init__() missing 1 required positional argument: 'group_id'
2023-09-11T20:45:43.414888491+02:00 
2023-09-11T20:45:43.414894816+02:00 During handling of the above exception, another exception occurred:
2023-09-11T20:45:43.414901222+02:00 
2023-09-11T20:45:43.414907222+02:00 Traceback (most recent call last):
2023-09-11T20:45:43.414917595+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/uvicorn/protocols/http/httptools_impl.py", line 436, in run_asgi
2023-09-11T20:45:43.414924612+02:00     result = await app(  # type: ignore[func-returns-value]
2023-09-11T20:45:43.414931257+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__
2023-09-11T20:45:43.414937143+02:00     return await self.app(scope, receive, send)
2023-09-11T20:45:43.414944222+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/fastapi/applications.py", line 276, in __call__
2023-09-11T20:45:43.414959258+02:00     await super().__call__(scope, receive, send)
2023-09-11T20:45:43.414965079+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/applications.py", line 122, in __call__
2023-09-11T20:45:43.414970616+02:00     await self.middleware_stack(scope, receive, send)
2023-09-11T20:45:43.414976160+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/middleware/errors.py", line 184, in __call__
2023-09-11T20:45:43.414981795+02:00     raise exc
2023-09-11T20:45:43.414987268+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/middleware/errors.py", line 162, in __call__
2023-09-11T20:45:43.414992763+02:00     await self.app(scope, receive, _send)
2023-09-11T20:45:43.414997925+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/middleware/gzip.py", line 24, in __call__
2023-09-11T20:45:43.415003283+02:00     await responder(scope, receive, send)
2023-09-11T20:45:43.415008649+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/middleware/gzip.py", line 44, in __call__
2023-09-11T20:45:43.415013958+02:00     await self.app(scope, receive, self.send_with_gzip)
2023-09-11T20:45:43.415019461+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
2023-09-11T20:45:43.415025582+02:00     raise exc
2023-09-11T20:45:43.415031034+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
2023-09-11T20:45:43.415036253+02:00     await self.app(scope, receive, sender)
2023-09-11T20:45:43.415041512+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
2023-09-11T20:45:43.415046954+02:00     raise e
2023-09-11T20:45:43.415052408+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
2023-09-11T20:45:43.415057639+02:00     await self.app(scope, receive, send)
2023-09-11T20:45:43.415062981+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/routing.py", line 718, in __call__
2023-09-11T20:45:43.415068303+02:00     await route.handle(scope, receive, send)
2023-09-11T20:45:43.415073919+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/routing.py", line 276, in handle
2023-09-11T20:45:43.415080117+02:00     await self.app(scope, receive, send)
2023-09-11T20:45:43.415087813+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/routing.py", line 66, in app
2023-09-11T20:45:43.415093658+02:00     response = await func(request)
2023-09-11T20:45:43.415099146+02:00   File "/app/mealie/routes/_base/routers.py", line 35, in custom_route_handler
2023-09-11T20:45:43.415104748+02:00     response = await original_route_handler(request)
2023-09-11T20:45:43.415110075+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/fastapi/routing.py", line 237, in app
2023-09-11T20:45:43.415115491+02:00     raw_response = await run_endpoint_function(
2023-09-11T20:45:43.415120779+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/fastapi/routing.py", line 165, in run_endpoint_function
2023-09-11T20:45:43.415126453+02:00     return await run_in_threadpool(dependant.call, **values)
2023-09-11T20:45:43.415132304+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
2023-09-11T20:45:43.415138046+02:00     return await anyio.to_thread.run_sync(func, *args)
2023-09-11T20:45:43.415143363+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/anyio/to_thread.py", line 31, in run_sync
2023-09-11T20:45:43.415148711+02:00     return await get_asynclib().run_sync_in_worker_thread(
2023-09-11T20:45:43.415155274+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
2023-09-11T20:45:43.415161441+02:00     return await future
2023-09-11T20:45:43.415166927+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 867, in run
2023-09-11T20:45:43.415172226+02:00     result = context.run(func, *args)
2023-09-11T20:45:43.415177490+02:00   File "/app/mealie/routes/recipe/recipe_crud_routes.py", line 226, in create_recipe_from_zip
2023-09-11T20:45:43.415182905+02:00     recipe = self.service.create_from_zip(archive, temp_path)
2023-09-11T20:45:43.415188144+02:00   File "/app/mealie/services/recipe/recipe_service.py", line 183, in create_from_zip
2023-09-11T20:45:43.415193875+02:00     recipe = self.create_one(Recipe(**recipe_dict))
2023-09-11T20:45:43.415199264+02:00   File "/app/mealie/services/recipe/recipe_service.py", line 146, in create_one
2023-09-11T20:45:43.415214391+02:00     new_recipe = self.repos.recipes.create(data)
2023-09-11T20:45:43.415222680+02:00   File "/app/mealie/repos/repository_recipes.py", line 36, in create
2023-09-11T20:45:43.415228351+02:00     return super().create(document)
2023-09-11T20:45:43.415233802+02:00   File "/app/mealie/repos/repository_generic.py", line 168, in create
2023-09-11T20:45:43.415239595+02:00     new_document = self.model(session=self.session, **data)
2023-09-11T20:45:43.415244863+02:00   File "<string>", line 4, in __init__
2023-09-11T20:45:43.415250291+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/sqlalchemy/orm/state.py", line 566, in _initialize_instance
2023-09-11T20:45:43.415255705+02:00     with util.safe_reraise():
2023-09-11T20:45:43.415260926+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 147, in __exit__
2023-09-11T20:45:43.415266338+02:00     raise exc_value.with_traceback(exc_tb)
2023-09-11T20:45:43.415271523+02:00   File "/opt/pysetup/.venv/lib/python3.10/site-packages/sqlalchemy/orm/state.py", line 564, in _initialize_instance
2023-09-11T20:45:43.415277268+02:00     manager.original_init(*mixed[1:], **kwargs)
2023-09-11T20:45:43.415282505+02:00   File "/app/mealie/db/models/recipe/api_extras.py", line 19, in wrapper
2023-09-11T20:45:43.415287761+02:00     return func(*args, extras=extras, **kwargs)
2023-09-11T20:45:43.415292914+02:00   File "/app/mealie/db/models/_model_utils/auto_init.py", line 187, in wrapper
2023-09-11T20:45:43.415298208+02:00     instances = handle_many_to_many(session, get_attr, relation_cls, val)
2023-09-11T20:45:43.415303554+02:00   File "/app/mealie/db/models/_model_utils/auto_init.py", line 73, in handle_many_to_many
2023-09-11T20:45:43.415308825+02:00     return handle_one_to_many_list(session, get_attr, relation_cls, all_elements)
2023-09-11T20:45:43.415314439+02:00   File "/app/mealie/db/models/_model_utils/auto_init.py", line 100, in handle_one_to_many_list
2023-09-11T20:45:43.415319952+02:00     new_elems = [safe_call(relation_cls, elem, session=session) for elem in elems_to_create]
2023-09-11T20:45:43.415325401+02:00   File "/app/mealie/db/models/_model_utils/auto_init.py", line 100, in <listcomp>
2023-09-11T20:45:43.415330654+02:00     new_elems = [safe_call(relation_cls, elem, session=session) for elem in elems_to_create]
2023-09-11T20:45:43.415336014+02:00   File "/app/mealie/db/models/_model_utils/helpers.py", line 44, in safe_call
2023-09-11T20:45:43.415341340+02:00     return func(**dict_args)
2023-09-11T20:45:43.415346581+02:00 TypeError: __init__() missing 1 required positional argument: 'group_id'
2023-09-11T20:45:45.861091056+02:00 INFO:     172.16.0.1:50292 - "GET /api/app/about HTTP/1.1" 200 OK
20 ```


### Mealie Version

https://github.com/hay-kot/mealie/commit/0a00a6ea0d05d2dd944dd9cc6713156dc767e1ed

### Deployment

Other

### Additional Deployment Details

Truenas scale

**Comment by zierbeek** [comment]
any idea?

**Comment by michael-genson** [comment]
Should be fixed by: https://github.com/mealie-recipes/mealie/pull/2585

**Comment by zierbeek** [comment]
> Should be fixed by: #2585

hero
